swagger_sdk_gen.sh

```diff
@@ -3,8 +3,8 @@ if $Build_Codegen ; then
 fi
 if [ "$Branch" = "snapshot" ]
   then
-    url="http://newapi-nightly.capillary.in/swagger.json"
-    version="http://newapi-nightly.capillary.in/version.json"
+    url="http://crm-nightly-new.cc.capillarytech.com/swagger.json"
+    version="http://crm-nightly-new.cc.capillarytech.com/version.json"
```

Comment:
version api is not working. correct url is - "https://api.crm-nightly-new.cc.capillarytech.com/version"